pipeline/MUSIC_DATA.csv

```diff
@@ -0,0 +1,19 @@
+,item_type,item_description,album_title,artist_name,country,amount_paid_usd,genres,release_date,sale_date
+0,t,Johanna in the 80s,None,Sweeney,Australia,6.5,"['adelaide', 'alternative', 'australian', 'electronic pop', 'queerpop', 'synthpop', 'Australia']",02-12-2024,03-12-2024
+1,a,Steve Rothery Band - Live in Frankfurt,None,Steve Rothery,United Kingdom,8.87,"['progressive rock', 'rock', 'Aylesbury']",30-11-2024,03-12-2024
+2,a,UFO 50,None,Eirik Suhrke,United States,10.0,"['chiptune', 'soundtrack', 'video game music', 'jazz', 'synth', 'Norway']",02-10-2024,03-12-2024
+3,a,A Fenestrated Fantasy,None,silk,France,1.9,"['punk', 'emo', 'emoviolence', 'screamo', 'New York']",01-12-2024,03-12-2024
+4,a,Hidden Treasures of Egypt,None,Skhemty,Spain,3.16,"['ambient', 'ancient egypt', 'dark ambient', 'dark dungeon music', 'dungeon synth', 'egyptian', 'egyptian ambient', 'egyptian music', 'egyptian synth', 'folklore', 'middle eastern', 'orchestral', 'oriental', 'pyramids', 'soundtrack', 'France']",01-12-2024,03-12-2024
+5,t,Thank You,Divine Essence,Svava,United States,1.29,"['a course in miracles', 'christian', 'devotional', 'inspirational', 'Duchesne']",04-03-2019,03-12-2024
+6,a,OCS LIVE AT PERMANENT ROADHOUSE,None,OCS,United Kingdom,7.0,"['folk', 'flok', 'kraut', 'psych', 'Los Angeles']",,03-12-2024
```

Comment:
I noticed this row is missing a date - could we either remove this row completely or replace with NULL?